### PR TITLE
added pipe function to spacy component

### PR DESCRIPTION
**Pull request by davidberenstein1957**

### CODE REVIEW

**Comment by LittlePea13** [comment]
Thank you so much for your PR, could you please add a brief description so I can check it and merge?

**Comment by davidberenstein1957** [comment]
A description into the .readme you mean? Or a function description? or a PR
description?

**Comment by LittlePea13** [comment]
I meant for the PR :)

File: spacy_component.py [hunk 1]
```diff
@@ -95,4 +94,25 @@ def set_annotations(self, doc: Doc, triplets: List[dict]):
...
         sentence_triplets = self._generate_triplets(doc.sents)
         self.set_annotations(doc, sentence_triplets)
-        return doc
\ No newline at end of file
+        return doc
+    
+    def pipe(self, stream, batch_size=128):
+        """
+        It takes a stream of documents, and for each document,
+        it assigns a score to each entity in the document
```
[the remark this answers is not in the document]
Comment (davidberenstein1957) [reply spacy_component.py]: i changed this

**Comment by davidberenstein1957** [comment]
@LittlePea13 I have updated the docstrings and readme, but I can't seem to find any way to add comments to the PR.

**Comment by LittlePea13** [comment]
Awesome, thanks. Thanks again for the PR :) I will merge now.

**Pull request merged**